assets/l10n/app_zh_TW.arb

```
@@ -0,0 +1,521 @@
+  "tagCocktail": "雞尾酒",
+  "tagTropicalCocktail": "熱帶雞尾酒",
+  "tagMateTea": "馬黛茶",
+  "tagMilk": "牛奶",
+  "tagKamaboko": "魚板",
+  "tagOden": "關東煮",
+  "tagCheese": "起司",
+  "tagEgg": "雞蛋",
+  "tagFriedEgg": "煎蛋",
+  "tagButter": "奶油",
+  "done": "完成",
+  "cancel": "取消",
```

RuboCop (1.81.7)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)